CONTRIBUTING.md

```diff
@@ -26,9 +29,63 @@ The Corporate CLA is for those who are contributing as a member of an organizati
 - [CLA for individuals](https://opensource.atlassian.com/individual)
 - [CLA for corporate contributors](https://opensource.atlassian.com/corporate)
 
+## Key Concepts
+
+### CSS-in-JS
+
+Compiled is a CSS-in-JS library: a library that lets React developers write CSS within their React code. Other libraries in this space include styled-components and Emotion. Below are some code examples.
+
+How people usually write in Compiled (we do want to change this long-term). This is similar to the styled function from [styled-components](https://styled-components.com/).
+
+```javascript
+import { styled } from '@compiled/react';
+
+const MyComponent = styled.div({
+  margin: 0,
+  color: 'blue',
+  padding: '250px',
+  backgroundColor: 'pink',
+});
+
+export const App = () => <MyComponent>Hello!</MyComponent>;
+```
+
+However, long term we want developers to write more like below. This is more similar to [emotion](https://emotion.sh/docs/css-prop).
+
+```javascript
+/** @jsx jsx */
+import { css } from '@compiled/react';
```

Comment:
```suggestion
/** @jsx jsx */
import { css, jsx } from '@compiled/react';
```

Ah this is probably more medium-term than long-term - we want this syntax due to the [new JSX transform](https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html) previously not being available in atlassian products. (I think our products were still on React 16? Whereas the new JSX transform is React 17+)

Long-term we can switch to the new JSX transform `/** @jsxAutomaticRuntime @compiled/react */` and avoid needing to import `jsx` from `@compiled/react`. @liamqma can you confirm whether we have existing plans for this?

Reply:
I am not aware of any plan in this area. Maybe Jake would know.

Reply:
ah no worries

Reply:
We 100% want to enable this. Just needs someone to champion it.